src/main/java/com/palantir/stash/stashbot/config/ConfigurationPersistenceImpl.java

```diff
@@ -387,7 +417,24 @@ private boolean getBoolean(HttpServletRequest req, String parameter) {
                 new DBParam("TIMESTAMPS_ENABLED", timestampJobOutputEnabled),
                 new DBParam("ANSICOLOR_ENABLED", ansiColorJobOutputEnabled),
                 new DBParam("BUILD_TIMEOUT_ENABLED", buildTimeoutSettings.getBuildTimeoutEnabled()),
-                new DBParam("BUILD_TIMEOUT", buildTimeoutSettings.getBuildTimeout())
+                new DBParam("BUILD_TIMEOUT", buildTimeoutSettings.getBuildTimeout()),
+                new DBParam("SLACK_ENABLED", slackSettings.getSlackEnabled()),
+                new DBParam("SLACK_TEAM_DOMAIN", slackSettings.getSlackTeamDomain()),
+                new DBParam("SLACK_AUTH_TOKEN", slackSettings.getSlackAuthToken()),
+                new DBParam("SLACK_BUILD_SERVER_URL", slackSettings.getSlackBuildServerUrl()),
+                new DBParam("SLACK_ROOM", slackSettings.getSlackRoom()),
+                new DBParam("SLACK_COMMIT_INFO_CHOICE", slackSettings.getSlackCommitInfoChoice()),
+                new DBParam("SLACK_INCLUDE_CUSTOM_MESSAGE", slackSettings.getSlackIncludeCustomMessage()),
+                new DBParam("SLACK_CUSTOM_MESSAGE", slackSettings.getSlackCustomMessage()),
+                new DBParam("SLACK_START_NOTIFICATION", slackSettings.getSlackStartNotification()),
+                new DBParam("SLACK_NOTIFY_SUCCESS", slackSettings.getSlackNotifySuccess()),
+                new DBParam("SLACK_NOTIFY_ABORTED", slackSettings.getSlackNotifyAborted()),
+                new DBParam("SLACK_NOTIFY_NOT_BUILT", slackSettings.getSlackNotifyNotBuilt()),
+                new DBParam("SLACK_NOTIFY_UNSTABLE", slackSettings.getSlackNotifyUnstable()),
+                new DBParam("SLACK_NOTIFY_FAILURE", slackSettings.getSlackNotifyFailure()),
+                new DBParam("SLACK_NOTIFY_BACK_TO_NORMAL", slackSettings.getSlackNotifyBackToNormal()),
+                new DBParam("SLACK_NOTIFY_REPEATED_FAILURE", slackSettings.getSlackNotifyRepeatedFailure()),
+                new DBParam("SLACK_INCLUDE_TEST_SUMMARY", slackSettings.getSlackIncludeTestSummary())
```

Comment:
I totally get the urge here but...oh god.  I'll check my enthusiasm level for this after I've taken a swing at the other issue but I suspect I'm gonna pass. :)